app/views/setup_instructions/index.html.erb

```diff
@@ -70,6 +70,7 @@
       </p>
       <p class="govuk-body">This must be done by someone in your organisation who has permission to sign off and procure services'.</p>
     </div>
+    <hr class="govuk-section-break--l govuk-section-break--visible">
   </div>
 </div>
-<%= link_to "Getting help", help_index_path, class: 'govuk-link' %>
+<%= link_to "If you have trouble setting up GovWifi, contact us", signed_in_new_help_path, class: 'govuk-link govuk-body' %>
```

Comment:
GDS style is to link on the action words only - [read an in-depth explanation](https://www.youtube.com/watch?v=dQw4w9WgXcQ) about why.

```suggestion
<p class="govuk-body">If you have trouble setting up GovWifi, <%= link_to "contact us", signed_in_new_help_path, class: 'govuk-link govuk-body' %>.</p>
```